main.js

```diff
@@ -0,0 +1,74 @@
+const choices = ["rock","paper","scissors"]
+let playerScore = 0;
+let computerScore = 0;
+
+function playerChoice (){
+    let input = prompt("Choose rock, paper, or scissors");
+    while (input == null){  
+        input = prompt("Choose rock, paper, or scissors");
+    }
+    input = input.toLowerCase();
+    let check = validateInput(input);
+    while (check == false){
+        input = prompt("Please choose rock, paper, or scissors");
+             while (input == null) {
+                 input = prompt("Choose rock, paper, or scissors");
+             }
+             
+         input = input.toLowerCase();
+         check = validateInput(input);
+    }
+    return input;
+}
+
+function computerChoice(){
+    return choices[Math.floor(Math.random()*3)];
+}
+
+function validateInput(choice){
+    return choices.includes(choice);
+}
+
+function playRound(){
+    const playerSelection = playerChoice();
+    const computerSelection = computerChoice();
+    console.log(playerSelection);
+    console.log(computerSelection);
+
+    if (playerSelection === computerSelection) {
+        console.log("The result is a tie!");
+    } else if ((playerSelection === "rock" && computerSelection === "scissors") ||
+               (playerSelection === "paper" && computerSelection === "rock") ||
+               (playerSelection === "scissors" && computerSelection === "paper")) {
+        ++playerScore;
```

Comment:
Recommend using `playerScore += 1` instead, long story short it helps enforce good coding habits (here's a discussion on it if you're interested: https://github.com/airbnb/javascript/issues/540)